### ISSUE TITLE
sprites, if added, should serve as a tutorial on how to play the game and use the different things(duh)

### ISSUE THREAD

**Issue by Dapperbot28** [mraof/Minestuck#222]
when i started messing around with minestuck, i had none to little clue to how to even chaptalogue an item, or alchemise stuff. it would be cool to have a sprite to explain it

**Comment by kirderf1** [comment]
Yep. My plan right now is to have a sort of instruction booklet for everything before entry, while the sprite will be of help for stuff after entry.

**Comment by Dweblenod** [comment]
This has (partially) been resolved with #726 and a few other QoL features, enough that I will close this!
